docs/admin_room.md

```diff
@@ -110,6 +110,17 @@ aware of your actions.
 You must invite the bridge bot into the Matrix room for this to work.
 
 
+### `!unlink`
+
+`!unlink !room:example.com irc.network.net #channel`
+
+*This command only works for moderators of a bridged Matrix room and bridge administrators*
+
+This command allows you to unlink a IRC channel from a room without using the HTTP provisioning API. For bridge
+administrators this command does NOT validate that you have permission to do this on the IRC channel so please
+take care to ensure that the IRC channel is aware of your actions.
```

Comment:
I'd just describe what it does rather than mentioning the provis API at all.

```suggestion
This command allows you to unlink a IRC channel from a room. Users are only able to remove links for rooms they are a moderator in (power level of 50 or greater). Administrators of the bridge are able to remove links from any room.
```